waspc/src/Wasp/Generator/WebAppGenerator.hs

```diff
@@ -51,6 +52,8 @@ genWebApp spec = do
   sequence
     [ genFileCopy [relfile|README.md|],
       genFileCopy [relfile|tsconfig.json|],
+      genSharedFileCopy [relfile|validators.js|] [relfile|scripts/validators.mjs|],
```

Comment:
Maybe it makes sense to wrap these two lines into `genEnvValidation` or something similar.